src/components/scenes/TransactionListScene.js

```diff
@@ -99,48 +100,15 @@ class TransactionListComponent extends React.PureComponent<Props, State> {
           sections={this.section(transactions)}
           renderItem={this.renderTransaction}
           renderSectionHeader={this.renderSectionHeader}
-          renderSectionFooter={() => <View style={{ height: this.props.theme.rem(0.5) }} />}
+          renderSectionFooter={() => <SectionFooter />}
```

Comment:
Remove use of in line declared functions

Reply:
Fixed